k8s/aat/common/ccd/data-store-api.yaml

```diff
@@ -25,7 +25,8 @@ spec:
         CCD_DM_DOMAIN: ^https?://(?:api-gateway\.preprod\.dm\.reform\.hmcts\.net|dm-store-aat\.service\.core-compute-aat\.internal(?::\d+)?)
         CCD_CONDITIONAL_APIS_CASE_ASSIGNED_USER_AND_ROLES_ENABLED: true
         CCD_S2S_AUTHORISED_SERVICES_CASE_USER_ROLES: aac_manage_case_assignment,fpl_case_service
-        DUMMY_RESTART_VAR: 1
+        DATA_STORE_S2S_AUTHORISED_SERVICES: am_role_assignment_service
```

Comment:
It just override chart's env variable. So, you might need to include all other values from here https://github.com/hmcts/ccd-data-store-api/blob/master/charts/ccd-data-store-api/values.yaml#L34